accesscontroltool-bundle/pom.xml

```diff
@@ -24,7 +24,7 @@
     <description>The OSGi bundle containing all classes of the Access Control Tool.</description>
     
     <properties>
-        <tomcat.el.version>9.0.104</tomcat.el.version>
+        <tomcat.el.version>9.0.107</tomcat.el.version>
```

Comment:
We can now upgrade to 10.1 (as we require Java 11): https://tomcat.apache.org/whichversion.html